src/views/home/home.js

```diff
@@ -79,12 +68,13 @@ export default {
     ListPodcast,
     recentPodcast,
   },
-  async created() {
+  async mounted() {
```

Comment:
Just to clarify, why are we using mounted instead ?

Reply:
Created will not wait for all of the async operations to complete before moving on to the next stage when making API call.